### PR TITLE
WIP: Chore: Update to bevy 0.13

### PR DESCRIPTION
**Pull request by mogambro**
I tried updating to bevy 0.13, it seemed straightforward, but sadly nothing renders, the screen stays black :/

Still opening this, because maybe it can still be useful

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,13 +17,14 @@ bevy = { version = "0.12", default-features = false, features = [
     "bevy_sprite",
     "multi-threaded",
     "png",
+    "wayland",
```
Comment (mogambro) [inline Cargo.toml:20]: Sorry, without this it doesn't run on my machine, feel free to remove it of course

File: src/render_pipeline/voxel_render.rs [hunk 2]
```diff
@@ -170,10 +172,9 @@ fn queue_custom(
                 entity,
                 pipeline,
                 draw_function: draw_custom,
-                distance: rangefinder
-                    .distance_translation(&mesh_instance.transforms.transform.translation),
                 batch_range: 0..1,
                 dynamic_offset: None,
+                asset_id: mesh_instance.mesh_asset_id,
```
Comment (mogambro) [inline src/render_pipeline/voxel_render.rs:177]: maybe this is the culprit?

File: src/render_pipeline/voxel_world.rs [hunk 3]
```diff
@@ -112,65 +112,65 @@ impl Plugin for VoxelWorldPlugin {
                 format: TextureFormat::Rgba8Unorm,
                 usage: TextureUsages::STORAGE_BINDING,
             },
+            TextureDataOrder::LayerMajor,
```
Comment (mogambro) [inline src/render_pipeline/voxel_world.rs:115]: Docs say this corresponds to the previous behaviour

**Comment by ria8651** [comment]
Not sure how I missed this pull request, thanks for the effort though. Closing now that #8 has updated to bevy 0.14.

**Pull request closed without merging**